sync/src/main/java/tech/pegasys/teku/sync/forward/multipeer/SyncController.java

```diff
@@ -145,6 +144,11 @@ public void unsubscribeFromSyncChanges(final long subscriberId) {
 
   private void notifySubscribers(final boolean syncing) {
     subscriberExecutor.execute(() -> subscribers.deliver(SyncSubscriber::onSyncingChange, syncing));
+    if (syncing) {
+      eventLogger.syncStart();
+    } else {
+      eventLogger.syncCompleted();
+    }
```

Comment:
@ajsutton i realised this is always called at the same time as notifySubscribers so suggested the eventLogger is inside the notify, but figured I'd make sure you're happy with that...

Reply:
Yeah I think that makes sense.  Almost tempting to just register it as a subscriber but I think that might be over engineering...

Reply:
lets keep it simple for now :)